templates/zerver/create_realm.html

```diff
@@ -4,10 +4,29 @@
 {# This is where we pitch the app and solicit signups. #}
 
 {% block portico_content %}
+<script src='https://www.google.com/recaptcha/api.js'></script>
 <script type="text/javascript">
 $(function () {
     common.autofocus('#email');
+    var submitButton = $("#send_confirm").find(":submit");
+    submitButton.click(validate);
 });
+
+function onSubmit () {
+    var response = grecaptcha.getResponse();
+    $('#g-recaptcha-response').text(response);
+    document.getElementById('send_confirm').submit();
+}
+
+function validate (event) {
+    var form = document.getElementById('send_confirm');
+    if (form.checkValidity()) {
+        grecaptcha.execute();
```

Comment:
This just breaks if the server doesn't have reCAPTCHA configured, right?

Or if it doesn't now, it will when we avoid pulling in the reCAPTCHA library at all in that case. So we need to make sure it just submits the form normally then.

Reply:
Yep.